frappe/desk/doctype/notification_log/notification_log.json

```diff
@@ -91,6 +91,12 @@
    "fieldname": "attachment_link",
    "fieldtype": "HTML",
    "label": "Attachment Link"
+  },
+  {
+   "fieldname": "link",
+   "fieldtype": "Data",
+   "hidden": 1,
+   "label": "Link"
```

Comment:
Why, this field is not in the `field_order` list
Also, modified date should be changed which is required to apply this changes on migrate

Reply:
@shariquerik Thanks for the note.

have pushed the changes